scalafmt-tests/src/test/resources/unit/Comment.stat

```diff
@@ -966,11 +966,13 @@ object a {
       user_id: String, dept_id: Option[Int], catg_grp_id: Option[Int], // for store only
       catg_id: Option[Int], subcatg_id: Option[Int], brand_id: Option[Int], order_date: String,
       item_id: Long
-  ) = SalesUserActivity(user_id, dept_id, catg_grp_id, catg_id, subcatg_id, brand_id,
-    Some(dummyBrand), 1f, 1f, item_id, tenant, inid, catalog, channel,
-    order_date.split("-")(0).toInt, order_date.split("-")(1).toInt, "v1", order_date)
+  ) = SalesUserActivity(
```

Comment:
here and in the next change: fold=true, cfg=true, dangle=true lead to dangling open and close.